de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/system/40-sw01_swat/5492760_cli_gh41-dist_Update_PickingTray_Locator.sql

```diff
@@ -0,0 +1,10 @@
+UPDATE M_PickingSlot ps
+SET M_Locator_ID = x.M_Locator_ID
+FROM
+	(
+		select l.M_Locator_ID, l.M_Warehouse_ID
+		from M_Locator l
+		where l.IsAfterPickingLocator = 'Y'
+		and l.IsActive = 'Y' 
+	) x
+WHERE x.M_Warehouse_ID = ps.M_Warehouse_ID and x.M_Warehouse_ID <> ps.M_Warehouse_ID;
```

Comment:
i think this where clause is wrong...maybe it's simpler/clearer to just see the "hardcoded" ID of the existing after-picking-locator